src/components/InjectedComponents/TypedMessageRenderer.tsx

```diff
@@ -62,6 +68,17 @@ export const DefaultTypedMessageTextRenderer = React.memo(function DefaultTypedM
     )
 })
 
+export const DefaultTypedMessageImageRenderer = React.memo(function DefaultTypedMessageImageRenderer(
+    props: TypedMessageRendererProps<TypedMessageImage>,
+) {
+    return renderWithMetadata(
+        props,
+        <Typography variant="body1" style={{ lineBreak: 'anywhere' }} data-testid="text_payload">
+            <RenderText text={props.message.content}></RenderText>
```

Comment:
This looks isn't quite right for me. TypedMessageImage should represent an image after the post decryption.

TypedMessage is describing what is _in_ the payload (that user wants to encrypt), it doesn't care about the payload format. Whatever plain text payload, link payload or image stego payload.

This component should absolutely render an image.